site/layouts/partials/social.html

```diff
@@ -1,18 +1,16 @@
-{{ "<!-- Twitter -->" | safeHTML }}
 <meta name="twitter:card" content="{{ if .IsHome }}summary_large_image{{ else }}summary{{ end }}">
 <meta name="twitter:site" content="@{{ .Site.Params.twitter }}">
 <meta name="twitter:creator" content="@{{ .Site.Params.twitter }}">
 <meta name="twitter:title" content="{{ .Title | markdownify }}">
 <meta name="twitter:description" content="{{ .Page.Params.description | default .Site.Params.description | markdownify }}">
-<meta name="twitter:image" content="{{ if .IsHome }}{{ .Site.Params.social_logo_path | absURL }}{{ else }}{{ .Site.Params.social_image_path | absURL }}{{ end }}">
+<meta name="twitter:image" content="{{ .Site.Params.social_image_path | absURL }}">
```

Comment:
Just so that we are on the same page, you mean to switch to one OG image?

Reply:
I think we're fine to have a single one? Open to feedback though!